app/src/main/res/values/strings.xml

```diff
@@ -29,4 +29,9 @@
     <string name="email_required">Email or username is required.</string>
     <string name="invalid_login_message">Invalid credentials. Please double-check your e-mail, username or password.</string>
     <string name="login_password_empty">Password is required.</string>
+    <string name="note_list_bar">Cactus Notes</string>
+    <string name="learn_kotlin">Learn kotlin</string>
+    <string name="lorem_note">Extensions functions are of a great help when creating a clean...</string>
+    <string name="any_notes">You don`t have any notes. Click below button to add one!</string>
```

Comment:
We should use apostrophe (') here instead of a backtick (\`). You may need to escape the apostrophe by leading it with a back-slash (\\).

`"You don\'t have..."`